### ISSUE TITLE
Inform users when derivative creation fails

### ISSUE THREAD

**Issue by awead** [samvera/hyrax#1192]
### Descriptive summary

We've had issues with certain image files not working with ImageMagick. In order to help the user understand what's going on, we've added a error message to the activity stream when a derivative job fails for any reason.

It's very basic, and would need some additional work, such as the ability to remove it (?) or clear the message once a derivative was successfully created. Another option would be to put these in notifications instead of activity.

### Rationale

It's better the give the user more information than none at all.

### Related work

https://github.com/psu-stewardship/scholarsphere/commit/19ca12989

**Comment by mjgiarlo** [comment]
Thanks for writing this up, @awead. I wonder if this might be adapted to use (mailboxer) notifications instead of activity streams. The former seem more likely to get a user's attention.

**Comment by cjcolvar** [comment]
:+1: to some user notification of failed derivative creation.  We have a progress bar in Avalon currently that will report errors if they happen.  We'll definitely need some feedback like described in this ticket in Hyrax.

**Comment by awead** [comment]
@mjgiarlo yes, mailboxer notifications is probably better because they can be dismissed.

**Comment by atz** [comment]
Agree we should fold this into leveraging two yet-unimplemented things:
1. Any kind of visibility for jobs pending/completed in aggregate or per-user
2. A more comprehensive notifications approach
